support-frontend/assets/pages/digital-subscription-landing/components/termsAndConditions.jsx

```diff
@@ -0,0 +1,16 @@
+// @flow
+import React from 'react';
+
+// styles
+import './digitalSubscriptionLanding.scss';
+
+const TermsAndConditions = () => (
+  <div className="hope-is-power__terms">
+    <div className="hope-is-power--centered">
+      <h3>Promotion terms and conditions</h3>
+      <p>Offer subject to availability. Guardian News and Media Ltd (&quot;GNM&quot;) reserves the right to withdraw this promotion at any time. Full annual promotion <a href="https://www.theguardian.com/info/2014/aug/06/guardian-observer-digital-subscriptions-terms-conditions">terms and conditions</a>.</p>
```

Comment:
Is this definitely the right url for this? I can't see any copy related to the promotion on it, normally promotions are on  subcribe.theguardian.com/p/[promocode]/terms

Reply:
I'm not clear about the correct url actually but thanks for highlighting this. I'll double check.

Reply:
Have updated with the right url and copy